core/indexer/elasticsearch_test.go

```diff
@@ -2,55 +2,21 @@ package indexer_test
 
 import (
 	"bytes"
-	"encoding/hex"
-	"fmt"
-	"math/big"
 	"net/http"
 	"net/http/httptest"
+	"strings"
 	"testing"
-	"time"
 
 	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/core/check"
 	"github.com/ElrondNetwork/elrond-go/core/indexer"
 	"github.com/ElrondNetwork/elrond-go/core/mock"
-	"github.com/ElrondNetwork/elrond-go/data"
 	"github.com/ElrondNetwork/elrond-go/data/block"
-	"github.com/ElrondNetwork/elrond-go/data/smartContractResult"
-	"github.com/ElrondNetwork/elrond-go/data/transaction"
 	"github.com/ElrondNetwork/elrond-go/logger"
-	"github.com/gin-gonic/gin/json"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
a subset of these tests are integration tests. Maybe move them inside integrationTests. The remaining + missing tests might use the mocks for easier testing.

Reply:
require is used in all tests now. i think these tests can remain here.

Reply:
Talking about the test functions within this file :D not about the imports, didn't know where to put the comment